model/src/cfd.rs

```diff
@@ -1325,7 +1325,9 @@ impl Cfd {
                 price: settlement.price,
             })
         } else {
-            self.fail_collaborative_settlement(anyhow!("Cannot complete collaborative settlement"))
+            self.fail_collaborative_settlement(anyhow!(
+                "CFD not eligible for collaborative settlement anymore"
+            ))
```

Comment:
🙃 It would be better if we returned an error from `can_settle_collaboratively` and used it here. 

Otherwise what you have to do is know what `can_settle_collaboratively` does in order to write this error message, which is harder and can get out of sync.

Reply:
I implemented this, but will open in separate PR because I should get more reviews given that I am flipping some logic :)

Reply:
https://github.com/itchysats/itchysats/pull/2652